flashinfer/testing/utils.py

```
@@ -1482,19 +1546,52 @@ def call_fn_with_rotation(buf_idx: int):
         g.replay()
     torch.cuda.synchronize()
 
+    # Determine how many graph replays can run back-to-back before triggering
+    # inter-replay power throttling (separate from intra-graph throttling above).
+    #
+    # Each graph replay packs num_iters_within_graph kernels back-to-back, so
+    # the *compute* time per replay is num_iters_within_graph × single_kernel_ms.
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)